quaint/src/ast/values.rs

```diff
@@ -34,11 +35,464 @@ where
     }
 }
 
+#[derive(Debug, Clone, PartialEq)]
+pub struct Value<'a> {
+    pub typed: ValueType<'a>,
+    pub native_column_type: Option<Cow<'a, str>>,
+}
+
+impl<'a> Value<'a> {
+    /// Creates a new 32-bit signed integer.
+    pub fn int32<I>(value: I) -> Self
+    where
+        I: Into<i32>,
+    {
+        ValueType::int32(value).into_value()
+    }
+
+    /// Creates a new 64-bit signed integer.
+    pub fn int64<I>(value: I) -> Self
+    where
+        I: Into<i64>,
+    {
+        ValueType::int64(value).into_value()
+    }
+
+    /// Creates a new 32-bit signed integer.
+    pub fn integer<I>(value: I) -> Self
+    where
+        I: Into<i32>,
+    {
+        ValueType::int32(value).into_value()
+    }
```

Comment:
Isn't this redundant? We have `int32` already, with exactly the same implementation.

Reply:
Yes, It's redundant, I think we are using aliases from integer to int32. I didn't go that far on the semantics. I can improve that if you think it's a blocker, tho'

Reply:
I don't think this is a blocker, but the smaller the API, the better imho. And if we have just identified this redundancy, we might as well say goodbye to that `integer` method (and related `is_integer` and `as_integer` ones) now that it's fresh in our memory :)
Not a blocker at all, though.

Reply:
Agreed :) Let me fix it.

Reply:
`is_integer` and `as_integer` cannot be removed. They match on both `ValueType::Int32` and `ValueType::Int64`

Reply:
Ok, my bad.